packages/cli/test/helpers/stubs/get.ts

```diff
@@ -0,0 +1,178 @@
+'use strict'
```

Comment:
This file is in a shared `test/helpers/stubs` folder.. should we either name this `org-access-get.ts` or put it in an `org-access` folder?

Reply:
I thought about that, but it's not only used by access commands. There are other commands that exist in orgs-v5 that use these stubs. And while right now it makes sense to use `org` to categorize these, once we remove the orgs-v5 plugin, none of these commands will be connected in any meaningful way to the concept of `orgs`.

Reply:
I'd be open to finding some other word to use to make this filename more specific, if you have ideas. I tried, but gave up. There's a lot in here.

Reply:
oh you're right, i didn't look closely at what things were actually being stubbed here. it's just.. a bunch of random `get` things that happen to be used in the org test. so yeah, leaving it as is makes sense. it probably should be organized differently? but it is what it is, so 👍 sticking with the generic `get` name.